src/test/run-pass/issue-29516.rs

```diff
@@ -11,6 +11,7 @@
 #![feature(optin_builtin_traits)]
 
 trait NotSame {}
+#[allow(auto_impl)]
```

Comment:
Why not use `auto trait NotSame` here?

Reply:
Mostly because I don't know if we ever test stage0 or not, I guess the answer is never.

Reply:
Well, I certainly don't, but maybe others do. Not a big thing, but I'd like to see at least one test using *just* the new syntax.

Reply:
Is [this test](https://github.com/leodasvacas/rust/blob/15f4b27568e764434ea35d53011f501b99e742a8/src/test/run-pass/auto-traits.rs) enough?

Reply:
I guess that's true. Sure, good enough.